bundles/org.openhab.binding.modbus.e3dc/README.md

```diff
@@ -0,0 +1,371 @@
+# E3DC Binding
+
+<img align="right" src="./doc/E3DC_logo.png"/>
+
+Integrates the Home Power Plants from E3/DC GmbH into openHAB. See [E3DC Website](https://www.e3dc.com/) to find more informations about the device.
+The Power Plant handles all your Electrical Energy Resources like Photovoltaic Producers, Battery Storage, Wallbox Power Supply, Household Consumption and even more.  
+E3DC devices are integrated into the Modbus Binding. 
+
+See chapter [Thing Configuration](#thing-configuration) how to set them up or check the [full example Things](#things) for manual setup. 
+
+
+## Supported Things
+
+First you need a Bridge which establishes the basic connection towards your E3DC device
+
+| Name                  | Thing Type ID | Description                                                                                          |
+|-----------------------|---------------|------------------------------------------------------------------------------------------------------|
+| E3DC Home Power Plant | e3dc          | Provides Power values, String Details, Emergency Power Status and general Information of your E3DC Home Power Plant    |
+| E3DC Wallbox          | e3dc-wallbox  | Provides your Wallbox Settings. Switches like "Sunmode" or "1-Phase Charging" can be changed     |
+
+
+## Discovery
+
+There's no discovery. Modbus registers are available for all devices. 
+
+
+## Thing Configuration
+
+As mentioned earlier in the [Binding Description](#e3dc-binding) the needed Things can be found in the **Modbus Binding** and have to be added manually without Discovery
+
+<img align="right" src="./doc/E3DC_Modbus_Settings.png"/>
+
+1. Create _Modbus TCP Bridge_ with matching Settings of your E3DC Device
+
+* IP Address
+* Device ID
+* Port ID
+
+2. Create _E3DC Home Power Plant_ and attach it to the previous installed _Modbus TCP Bridge_. Configuration requires an approriate Data Refresh Interval with more than 1000 Milliseconds
+
+3. If you have a Wallbox attached add _E3DC Wallbox_ Thing with your previous installed _E3DC Home Power Plant_ as Bridge. Configuration requires a Wallbox ID between 0 and 7.
+
+Check the [full example Things](#things) for manual setup.
+
+### Modbus TCP Slave 
+
+| Parameter       | Type    | Description                                                             |           
+|-----------------|---------|-------------------------------------------------------------------------|
+| host            | text    | IP Address of your device                                               |
+| port            | integer | Modbus Port of your E3DC device Modbus Settings.. Default is 502 |
```

Comment:
```suggestion
| port            | integer | TCP Port of your E3DC device Modbus Settings. Default is 502 |
```

Reply:
oh man - so sorry for all these documentation errors. 
Fixed

Reply:
Never mind!